submissions/zhenyakornilov/js-dom/css/styles.css

```diff
@@ -0,0 +1,420 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+li {
+  list-style: none;
+}
+
+a,
+a:link,
+a:visited,
+a:hover,
+a:active,
+a:focus {
+  color: inherit;
+  text-decoration: none;
+  outline: none;
+}
+
+button {
+  background: none;
+  color: inherit;
+  border: none;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+:root {
```

Comment:
I change html font-size since I use **rem** and when html font-size is equal to 62.5% then 1 rem is like 10 pixels, 2.1rem is 21 pixels etc. It makes more clear to me rem values to interpret them as pixels.